xcode/Subconscious/Shared/Models/RecoveryPhrase.swift

```diff
@@ -11,26 +11,17 @@ import Foundation
 public struct RecoveryPhrase:
     Hashable,
     Equatable,
-    Identifiable,
-    Codable,
-    LosslessStringConvertible,
-    CustomDebugStringConvertible {
```

Comment:
RecoveryPhrase should not be Codable, LosslessStringConvertible, or CustomDebugStringConvertible, since we don't ever want it serialized.

We don't need it to be identifiable either. Hashable and Equatable is enough.